pkg/printers/internalversion/printers.go

```diff
@@ -332,6 +332,15 @@ func AddHandlers(h printers.PrintHandler) {
 	h.TableHandler(horizontalPodAutoscalerColumnDefinitions, printHorizontalPodAutoscaler)
 	h.TableHandler(horizontalPodAutoscalerColumnDefinitions, printHorizontalPodAutoscalerList)
 
+	verticalPodAutoscalerColumnDefinitions := []metav1beta1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "Selector", Type: "string", Description: autoscalingv2beta1.VerticalPodAutoscalerSpec{}.SwaggerDoc()["selector"]},
+		{Name: "UpdateMode", Type: "string", Description: autoscalingv2beta1.PodUpdatePolicy{}.SwaggerDoc()["updateMode"]},
```

Comment:
Selector, and even UpdateMode could be priority 1. `kubectl get` is useful when it prints the current status of a resource. Is there something in vpa status that you can add here? For example, deployment printer prints its desired and current replicas; pod printer prints its current ready containers and restart counts.